src/cpu/aarch64/brgemm/brgemm_types.hpp

```diff
@@ -191,6 +191,8 @@ struct brgemm_t {
     int LDB = 0;
     int LDC = 0;
     int LDD = 0;
+
+    int M, K, N;
```

Comment:
`M`, `K`, and `N` are literally declared above as `bcast_dim`, `reduce_dim` and `load_dim`. Any specific reason to add duplicated entries for them?

Reply:
Hi @dzarukin ,
The bcast, reduce and load dimensions are blocked sizes of M,K and N. M, K and N are actual sizes of the original matrix.